service/test/controllers/LearnerControllerTest.java

```diff
@@ -1,32 +1,80 @@
 package controllers;
 
+import static controllers.TestUtil.mapToJson;
 import static org.junit.Assert.assertEquals;
+import static org.powermock.api.mockito.PowerMockito.when;
 import static play.test.Helpers.route;
 
+import akka.actor.ActorRef;
+import akka.actor.ActorSystem;
+import akka.actor.Props;
 import com.fasterxml.jackson.databind.JsonNode;
+
+import java.io.File;
 import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+
+import modules.OnRequestHandler;
+import modules.StartModule;
+import org.junit.Assert;
+import org.junit.Before;
 import org.junit.Ignore;
 import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.core.classloader.annotations.SuppressStaticInitializationFor;
+import org.powermock.modules.junit4.PowerMockRunner;
 import org.sunbird.common.models.util.JsonKey;
 import org.sunbird.common.responsecode.ResponseCode;
+import play.Application;
+import play.Mode;
+import play.inject.guice.GuiceApplicationBuilder;
 import play.libs.Json;
 import play.mvc.Http;
 import play.mvc.Http.RequestBuilder;
 import play.mvc.Result;
 import play.test.Helpers;
+import util.RequestInterceptor;
 
 /** @author arvind */
-@Ignore
-public class LearnerControllerTest extends BaseControllerTest {
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({ RequestInterceptor.class})
+@SuppressStaticInitializationFor({"util.AuthenticationHelper", "util.Global"})
```

Comment:
util.Global doesn't exists anymore, we do not need to suppress it's initialization